### DISCUSSION TITLE
EU Digital COVID Certificate

### DISCUSSION THREAD

**Discussion by qwerty74M25** [mebjas/html5-qrcode#282] [Q&A]
I'm trying to read QRCode present in the EU Digital COVID Certificate but with no luck.
Is it too long ?
![green-pass jpg](https://user-images.githubusercontent.com/89469407/130639680-78cb7434-7894-446a-ab85-3d83c012f903.jpg)

**Comment by ROBERT-MCDOWELL** [comment]
well, it's better like that I think...

**Reply by trasherdk** [reply]
Dude :+1:
